geomstats/geometry/pre_shape.py

```diff
@@ -378,6 +379,339 @@ def integrability_tensor(self, tangent_vec_a, tangent_vec_b, base_point):
 
         return vertical + horizontal_
 
+    def integrability_tensor(self, tangent_vec_x, tangent_vec_e, base_point):
+        r"""Compute the fundamental tensor A of the submersion.
+
+        The fundamental tensor A is defined for tangent vectors of the total
+        space by [O'Neill]_
+        :math: `A_X Y = ver\nabla^M_{hor X} (hor Y)
+            + hor \nabla^M_{hor X}( ver Y)`
+        where :math: `hor, ver` are the horizontal and vertical projections.
+
+        For the Kendall shape space, we have the closed-form expression at
+        base-point P:
+        :math: `A_X E = P Sylv_P(E^\top hor(X)) + F + <F,P> P` where  :math:
+        `F = hor(X) Sylv_P(P^\top E)` and :math: `Sylv_P(B)` is the
+        unique skew-symmetric matrix :math: `\Omega` solution of
+        :math: `P^\top P \Omega + \Omega P^\top P = B - B^\top`.
+
+        Parameters
+        ----------
+        tangent_vec_x : array-like, shape=[..., {ambient_dim, [n, n]}]
+            Tangent vector at `base_point`.
+        tangent_vec_e : array-like, shape=[..., {ambient_dim, [n, n]}]
+            Tangent vector at `base_point`.
+        base_point : array-like, shape=[..., {ambient_dim, [n, n]}]
+            Point of the total space.
+
+        Returns
+        -------
+        vector : array-like, shape=[..., {ambient_dim, [n, n]}]
+            Tangent vector at `base_point`, result of the A tensor applied to
+            `tangent_vec_x` and `tangent_vec_e`.
+
+        References
+        ----------
+        [O'Neill]  O’Neill, Barrett. The Fundamental Equations of a Submersion,
```

Comment:
The syntax for a bib entry is `.. [key]` (two dots in front of the square bracket`

Reply:
Is there a description of this syntax? I could not spot it.

Reply:
Yes, here for example:
https://github.com/geomstats/geomstats/blob/65e7a54d970897b3dc31eb5ca0933cbfd5d87607/geomstats/geometry/lie_algebra.py#L55